assets/app/components/AddSite/TemplateSiteList/templateSite.js

```diff
@@ -100,6 +100,12 @@ class TemplateSite extends React.Component {
                 type="text"
                 value={this.state.siteName}
                 onChange={this.handleChange} />
+              <label for="site-org">Repository Location</label>
+              <input
+                name="site-org"
+                type="text"
+                value={this.state.orgName}
+                onChange={this.handleChange} />
```

Comment:
Currently handleChange isn't generic, it is specifically keyed to update the 'siteName' property of the component's state. The method would have to be changed to allow it to be used with different form fields